### PR TITLE
Chase

### PR DESCRIPTION
**Pull request by chase-abbott**

### CODE REVIEW

File: local-storage-utils.js [hunk 1]
```diff
@@ -34,6 +35,57 @@ export function createUser(name, tagline, formArray) {
     return user
```
Comment (NickDayFSD) [inline local-storage-utils.js]: add semi colon

File: map/map.js [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
+const user = getUser();
 const canvas = document.querySelector('#main-canvas');
 const ctx = canvas.getContext('2d');
 canvas.width = 1200;
 canvas.height = 900;
 
+
+if (areQuestsCompleted(user)) {
+    window.location = '../results';
+};
```
Comment (NickDayFSD) [inline map/map.js]: remove semi

File: quest/quest.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 const params = new URLSearchParams(window.location.search);
 import { quests } from './data.js';
-import { setUser, getUser } from '../local-storage-utils.js';
-import { findById, userSuccess } from '../utils.js';
+import { setUser, getUser, poistiveUserUpdate, negativeUserUpdate } from '../local-storage-utils.js';
```
Comment (NickDayFSD) [inline quest/quest.js]: remove setUser

File: quest/quest.js [hunk 4]
```diff
@@ -14,27 +14,27 @@ const descriptionContainer = document.getElementById('description');
...
-const questId = params.get('id')
-const quest = findById(quests, questId)
+const questId = params.get('id');
+const quest = findById(quests, questId);
 
 h2.textContent = quest.title;
 descriptionContainer.textContent = quest.description;
 
-
 image.src = `../assets/${quest.image}`
```
Comment (NickDayFSD) [inline quest/quest.js]: add semi-colon

File: quest/quest.js [hunk 5]
```diff
@@ -47,39 +47,20 @@ form.addEventListener('submit', (e) => {
...
 
     submitButton.style.display = 'none';
 });
 
 resultsButton.addEventListener('click', () => {
     window.location = '../map';
-
-})
\ No newline at end of file
+})
```
Comment (NickDayFSD) [inline quest/quest.js]: add semi-colon

File: test/local-storage-utils.test.js [hunk 6]
```diff
@@ -40,3 +40,20 @@ test('Test getUser function', (expect) => {
...
+
+test('Test to see if all user quests are completed', (expect) => {
+    const expected = false;
+
+    const user = {
+        completedQuests: {
+            beast: true,
+            fugitive: false,
+        }
+    }
```
Comment (NickDayFSD) [inline test/local-storage-utils.test.js]: add semi-colon

File: test/utils.test.js [hunk 7]
```diff
@@ -1,13 +1,14 @@
-import { rollGenerator, findById, userSuccess, luckRoll } from '../utils.js';
+import { rollGenerator, findById, userSuccess, luckRoll, doesUserSatisfyRequirements } from '../utils.js';
```
Comment (NickDayFSD) [inline test/utils.test.js]: remove userSuccess, not being used

**Review by NickDayFSD — CHANGES_REQUESTED** [review]
Check  comments for changes

**Review by NickDayFSD — APPROVED** [review]
looks groovy

**Review by David-Del1 — APPROVED** [review]

**Pull request merged**